blockchain/util/insert_db.py

```diff
@@ -40,7 +40,7 @@
 
 
 def load_required_nodes():
```

Comment:
If I'm right, this is only being called by itself - should this be moved to `scripts` and made to accept CLI parameters?

Is that already a ticket?